CHANGELOG.md

```diff
@@ -4,6 +4,25 @@ All notable changes to this project will be documented in this file.
+* Service: Restrict node operations to administrators. [#1779](https://github.com/microsoft/onefuzz/pull/1779)
+
+### Changed
+* CLI/Service: Updated multiple first-party and third-party Python dependencies. [#1784](https://github.com/microsoft/onefuzz/pull/1784)
+* Agent/Supervisor/Proxy: Updated multiple third-party Rust dependencies. [#1619](https://github.com/microsoft/onefuzz/pull/1619), [#1644](https://github.com/microsoft/onefuzz/pull/1644), [#1645](https://github.com/microsoft/onefuzz/pull/1645), [#1646](https://github.com/microsoft/onefuzz/pull/1646), [#1655](https://github.com/microsoft/onefuzz/pull/1655), [#1700](https://github.com/microsoft/onefuzz/pull/1700), [#1849](https://github.com/microsoft/onefuzz/pull/1849), [#1882](https://github.com/microsoft/onefuzz/pull/1882)
+* Agent: Separate `llvm-symbolizer` setup from sanitizer environment variable initialization. [#1778](https://github.com/microsoft/onefuzz/pull/1778)
+* Agent: Set the TSan options based on the external symbolizer. [#1787](https://github.com/microsoft/onefuzz/pull/1787)
+* Service: Removed the `client_secret` from the configuration file to eliminate the potential for the secret to be persisted. If you have persisted a client secret in your configuration file, remove it. [#1918](https://github.com/microsoft/onefuzz/pull/1918)
+* Service: Use a sas url to download log files. [#1920](https://github.com/microsoft/onefuzz/pull/1920)
```

Comment:
```suggestion
* CLI: Use a sas url to download log files. [#1920](https://github.com/microsoft/onefuzz/pull/1920)
```

Reply:
```suggestion
* CLI: Use a SAS URL to download log files. [#1920](https://github.com/microsoft/onefuzz/pull/1920)
```
(apply to any other use of acronyms like "SAS", "URL", "CLI")